joomla-projects/gsoc18_override_management
# Final Product

Pull Request for Issue #38 

### Summary of Changes
Features added in Improve Override Management

**1. Core and Diff view** 
![screenshot from 2018-07-17 14-42-55](https://user-images.githubusercontent.com/30978328/42808002-e130d52a-89cf-11e8-8973-1314d759cf1b.png)

**2. Updated override history list.**
![screenshot from 2018-07-17 14-43-37](https://user-images.githubusercontent.com/30978328/42808025-ed5f7a54-89cf-11e8-9e73-abafb943d5c1.png)

**3. Quick icon notification plugin.**
![screenshot from 2018-07-17 14-05-12](https://user-images.githubusercontent.com/30978328/42808169-418aaa18-89d0-11e8-925a-f4ce7c89e8fb.png)

**Installer override plugin must  enabled to fetch result. And show in quick icon.** 
![screenshot from 2018-07-17 01-55-56](https://user-images.githubusercontent.com/30978328/42808242-71dff3a8-89d0-11e8-877b-762b3da62a39.png)


**4. Override control plugin.**
![screenshot from 2018-07-17 01-56-38](https://user-images.githubusercontent.com/30978328/42808170-418eca3a-89d0-11e8-9ce7-f646313cb329.png)
![screenshot from 2018-07-17 14-47-26](https://user-images.githubusercontent.com/30978328/42808206-5b0c4974-89d0-11e8-843a-c0f18ec530d7.png)

### Testing Instructions
**1. Create override from this override list from create override tab.**
![screenshot from 2018-07-17 14-54-34](https://user-images.githubusercontent.com/30978328/42808659-641b863c-89d1-11e8-8e77-240485cc8de8.png)

2. Then, try to update you extension for which you created the override by any of them.
   A. Extension Install
   B. Extension Update
   C. Joomla Update

3. If any override core files is got an update the you will receive message after update. And, when  you visit control panel you will see quick icon response after update.

4. Then, visit to your template settings you will see. A tab with updated files which contains list of updated file with belongs to current template.

**Note**: During this project new feature is introduced. 
To support more than two form in one page with different id 
PR for this https://github.com/joomla-projects/gsoc18_override_management/pull/36

### Documentation Changes Required
Yes
